.github/workflows/github-actions-ci.yml

```diff
@@ -59,6 +59,7 @@ jobs:
       - name: Install
         run: |
           conda activate test-env
+          conda install -c conda-forge pytables
```

Comment:
Is this because pytables doesn't work on windows with python 3.9? This is only being conda installed for tests, so I imagine if it's needed it needs be installed this way through setup.py so that it is correctly installed when allensdk is installed. It looks like possibly another strategy mentioned in [this thread](https://github.com/PyTables/PyTables/issues/933) is to set an environment variable `CONDA_DLL_SEARCH_MODIFICATION_ENABLE=1` if that is the issue you were trying to get around.

Reply:
I'll give it a shot and see if that also fixes it.

Reply:
That seems to have worked? I assume though that you were thinking of adding the environment variable to `setup.py` instead so that the install just works, correct?

Reply:
Changed to the `setup.py` option. Looks to have passed all tests. Additionally, I tested this with a fresh miniconda environment and found no issues with install or testing.